### PR TITLE
Update "How it works" section and add some articles about Dangerzone

### PR DESCRIPTION
**Pull request by apyrgio**
Update the "How it works" section, since it was out of sync with the recent developments on Dangerzone. Also, add the results of our latest security audit, as well as some articles from journalism-related media, which mention the use of Dangerzone.

See also freedomofpress/dangerzone#807

### CODE REVIEW

**Comment by apyrgio** [comment]
Heads up, I've converted the changes in this PR from HTML to Markdown, to make them work for 11ty.

File: src/about.md [hunk 1]
```diff
@@ -109,6 +106,14 @@ If you keep Docker Desktop updated and regularly update the container that Dange
 
 Another way a malicious document may harm your system, even with Dangerzone, is if it is crafted to attack the document previewing capabilities of the operating system itself (e.g. the part that generates file thumbnails or document previews in a side-panel of the file manager). Due to the high level of integration of these features in the operating system, disabling them completely may be challenging. For this reason, keeping your system always up to date is the most practical solution to minimize this risk.
 
+While we are doing our best to inform journalists about these risks and keep them as safe as possible, we believe its important for third-parties to independently assess our assumptions. For this reason, Dangerzone underwent its [first security audit](https://freedom.press/news/dangerzone-receives-favorable-audit)</a> on December 2023 by [Include Security](https://includesecurity.com/) with support from the [Open Technology Fund](https://www.opentech.fund/). The audit was generally favorable, as it didn't identify any high-risk findings, except for 3 low-risk and 7 informational findings.
```
Comment (eloquence) [inline src/about.md:109]: its important->it's important

third-parties->third parties

except for 3 low-risk and 7 informational findings. -> (it identified 3 low-risk and 7 informational findings) 

(The "except for" is not in fact an exception to the previous statement.)

File: src/about.md [hunk 2]
```diff
@@ -109,6 +106,14 @@ If you keep Docker Desktop updated and regularly update the container that Dange
...
+
+What others have written about Dangerzone
+-----------------------------------------
+
+* [GIJN Toolbox: Cutting-Edge — and Free — Online Investigative Tools You Can Try Right Now](https://gijn.org/stories/cutting-edge-free-online-investigative-tools/)
+* [When security matters: working with Qubes OS at the Guardian](https://www.theguardian.com/info/2024/apr/04/when-security-matters-working-with-qubes-os-at-the-guardian)
```
Comment (eloquence) [inline src/about.md:115]: I'd add dates/authors for these, so the list can be ordered reverse chron as it gets longer.

**Review by eloquence — CHANGES_REQUESTED** [review]
A couple of nits but mostly LGTM! Should we wait with merging this until https://github.com/freedomofpress/dangerzone/pull/748 lands? AIUI this describes the post-748 architecture.

File: src/about.md (hunk 1 above)
Comment (apyrgio) [reply src/about.md]: Thanks for the fixes Erik. I've changed this paragraph in 57257c51b45ee876cbaa08fca3677bda84b4236e.

File: src/about.md (hunk 2 above)
Comment (apyrgio) [reply src/about.md]: Makes sense to me. I've taken a stab at c5bbd4aaca0650695865767204b08159bab3ecea using italics for the extra data (else it looked a bit off to me...). Tell me if you have a better idea!

**Comment by apyrgio** [comment]
> Should we wait with merging this until https://github.com/freedomofpress/dangerzone/pull/748 lands? AIUI this describes the post-748 architecture.

Actually, with this PR we describe the architecture we have currently (minus the gVisor parts which will be covered in #41). We still refer to the second container in the section of how Dangerzone works, and the fact that we mount the pixel data in it. Once we merge freedomofpress/dangerzone#748, we'll have to update it once more.

**Review by eloquence — APPROVED** [review]
LGTM!

**Pull request merged**